### PR TITLE
[node-cache_v4.x.x] Add libdefs for node-cache

### PR DESCRIPTION
**Pull request by nvie**

### CODE REVIEW

File: definitions/npm/node-cache_v4.x.x/flow_v0.25.x-/node-cache_v4.x.x.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+|};
+
+declare module "node-cache" {
+  declare class NodeCache {
+    constructor(options?: nodeCache$Options): NodeCache;
+    get(key: string): mixed | void;
+    set(key: string, val: mixed, ttl?: number): boolean;
+  }
+
+  declare module.exports: Class<NodeCache>;
```
Comment (villesau) [inline definitions/npm/node-cache_v4.x.x/flow_v0.25.x-/node-cache_v4.x.x.js]: same as in https://github.com/flow-typed/flow-typed/pull/3047#discussion_r245523167, you can make the types better scoped using `declare export`

Comment (nvie) [reply definitions/npm/node-cache_v4.x.x/flow_v0.25.x-/node-cache_v4.x.x.js]: Thanks for the feedback – addressed in 9cb06b1d680f617de27d739842c229b147627259! 🙏

Comment (nvie) [reply definitions/npm/node-cache_v4.x.x/flow_v0.25.x-/node-cache_v4.x.x.js]: Thanks – this is fantastic feedback.  It's been really confusing to understand the implications of these statements, but this really helps.  I've addressed this in bee7bb2377bff62172b1a240335915c58c74cb1b now – does this look better now?

File: definitions/npm/node-cache_v4.x.x/flow_v0.25.x-/node-cache_v4.x.x.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
+// @flow strict
```
Comment (villesau) [inline definitions/npm/node-cache_v4.x.x/flow_v0.25.x-/node-cache_v4.x.x.js]: Although I don't think it makes any harm, `// @flow strict` is not actually needed in libdefs.

Comment (nvie) [reply definitions/npm/node-cache_v4.x.x/flow_v0.25.x-/node-cache_v4.x.x.js]: Cheers, good to know 👍 — fixed in 572f99a

**Pull request merged**